Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1293,6 +1293,11 @@ void RenderLayerScrollableArea::updateScrollbarsAfterStyleChange(const RenderSty
 void RenderLayerScrollableArea::updateScrollbarsAfterLayout()
 {
     RenderBox* box = m_layer.renderBox();
```

Comment:
I know this was already not using smart pointers before your change, but it would be nice to update it:
```suggestion
    CheckedPtr box = m_layer.renderBox();
```